testing/liberasurecode/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: Ehud Kaldor <ehud@unfairfunction.org>
+# Maintainer: Ehud Kaldor <ehud@unfairfunction.org>
+pkgname=liberasurecode
+pkgver=1.5.0
+pkgrel=0
+pkgdesc="Erasure Code API library written in C with pluggable Erasure Code backends"
+url="https://github.com/openstack/liberasurecode"
+arch="all"
+license="BSD-2-Clause" 
+depends=""
+makedepends="libtool automake autoconf zlib-dev"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="${pkgname}-${pkgver}-src.tar.gz::https://github.com/openstack/liberasurecode/archive/$pkgver.tar.gz"
+builddir="$srcdir"/$pkgname-$pkgver 
+
+build() {
+	cd "$builddir"
+	./autogen.sh
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var
+	make
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" install
+	install -Dm644 COPYING "$pkgdir"/usr/share/licenses/$pkgname/COPYING
```

Comment:
This is just text of the BSD-2-Clause license without any additional information, so including it in the package is unnecessary.